filebeat/input/filestream/config.go

```diff
@@ -142,3 +144,57 @@ func (c *config) Validate() error {
 
 	return nil
 }
+
+// ValidateInputIDs checks all filestream inputs to ensure all have an ID and
+// the ID is unique. If there is any empty or duplicated ID, it logs an error
+// containing the offending input configurations and returns an error containing
+// the duplicated IDs.
```

Comment:
This description is not true anymore because we allow empty IDs.

```suggestion
// ValidateInputIDs checks all filestream inputs to ensure all input IDs are unique.
// If there is a duplicated ID, it logs an error containing the offending input
// configurations and returns an error containing the duplicated IDs.
// An empty ID is treated as a valid ID value. Multiple empty IDs are not unique.
```